Magento2/Sniffs/Functions/FunctionsDeprecatedWithoutArgumentSniff.php

```diff
@@ -31,15 +31,15 @@ class FunctionsDeprecatedWithoutArgumentSniff implements Sniff
     private const WARNING_CODE = 'FunctionsDeprecatedWithoutArgument';
 
     /**
-     * Deprecated functions without argument.
+     * Deprecated functions without argument
```

Comment:
Let's specify the source of data
```suggestion
     * Deprecated functions without argument https://wiki.php.net/rfc/deprecations_php_8_1
```